fluidBuild.config.cjs

```diff
@@ -88,10 +97,22 @@ module.exports = {
 		// therefore we need to require both before running api-extractor.
 		"check:release-tags": ["tsc", "build:esnext"],
 		"check:are-the-types-wrong": ["build"],
+		"check:format": {
+			dependsOn: [...checkFormattingTasks],
+			script: false,
+		},
+		"format": {
+			dependsOn: ["format:prettier", "format:biome"],
```

Comment:
Nit: should these be extracted into a `formatTasks` constant like `checkFormattingTasks` for consistency?

Reply:
We could, but it's not clearer IMO. This combination is only used once (here), and the task definition is "format", so it seems clear these are the format tasks. The constants are only used in cases where we need to use a particular combination in more than one place.